support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -92,19 +107,113 @@ function PaperLandingPage({
 					</Block>
 				</CentredContainer>
 			</FullWidthContainer>
+
 			<FullWidthContainer theme="dark" hasOverlap>
 				<CentredContainer>
-					<Prices
+					<PaperProductPrices
+						productPrices={productPrices}
+						tab={selectedTab}
+						setTabAction={setSelectedTab}
+					/>
+				</CentredContainer>
+			</FullWidthContainer>
+		</Page>
+	);
+}
+
+function PaperLandingPageVariant({
+	productPrices,
+	promotionCopy,
+	isPriceCardsAbTestVariant,
+}: PaperLandingContentPropTypes) {
+	const sanitisedPromoCopy = getPromotionCopy(promotionCopy);
+	const fulfilment: PaperFulfilmentOptions = window.location.pathname.includes(
+		'delivery',
+	)
+		? HomeDelivery
+		: Collection;
+	const [selectedTab, setSelectedTab] =
+		useState<PaperFulfilmentOptions>(fulfilment);
+
+	if (!productPrices) {
+		return null;
+	}
+
+	function handleSetTabAction(newTab: PaperFulfilmentOptions) {
+		setSelectedTab(newTab);
+		sendTrackingEventsOnClick({
+			id: `Paper_${newTab}-tab`,
+			// eg. Paper_Collection-tab or Paper_HomeDelivery-tab
+			product: 'Paper',
+			componentType: 'ACQUISITIONS_BUTTON',
+		})();
+		window.history.replaceState({}, '', paperSubsUrl(newTab === HomeDelivery));
+	}
+
+	return (
+		<Page
+			id={pageQaId}
+			header={<Header countryGroupId={GBPCountries} />}
+			footer={paperSubsFooter}
+		>
+			<FullWidthContainer cssOverrides={paperHeroContainerOverrides}>
+				<PriceCardsPaperHero
+					productPrices={productPrices}
+					promotionCopy={sanitisedPromoCopy}
+				/>
+			</FullWidthContainer>
+			<FullWidthContainer theme="dark">
+				<CentredContainer>
+					<PaperProductPrices
 						productPrices={productPrices}
 						tab={selectedTab}
 						setTabAction={setSelectedTab}
+						isPriceCardsAbTestVariant={isPriceCardsAbTestVariant}
 					/>
 				</CentredContainer>
 			</FullWidthContainer>
+			<FullWidthContainer cssOverrides={paperHeroContainerOverrides}>
+				<CentredContainer>
+					<PaperProductInfo promotionCopy={sanitisedPromoCopy} />
+				</CentredContainer>
+			</FullWidthContainer>
+			<FullWidthContainer>
+				<CentredContainer>
+					<Block>
+						<div css={tabsTabletSpacing}>
+							<Tabs
+								selectedTab={selectedTab}
+								setTabAction={handleSetTabAction}
+							/>
+						</div>
+					</Block>
+				</CentredContainer>
+			</FullWidthContainer>
 		</Page>
 	);
 }
 
+function PaperLandingPage({
+	productPrices,
+	promotionCopy,
+	participations,
+}: PaperLandingPropTypes) {
+	const isPriceCardsAbTestVariant =
+		participations.newspaperPriceCards === 'variant';
+	return isPriceCardsAbTestVariant ? (
+		<PaperLandingPageVariant
+			productPrices={productPrices}
+			promotionCopy={promotionCopy}
+			isPriceCardsAbTestVariant={isPriceCardsAbTestVariant}
```

Comment:
Not sure here, think I need this.  Its a parameter used in PaperLandingPageVariant and omitted from PaperLandingPageControl. Both of these components call the SAME PaperProductPrices component which needs to decide whether you coming from Control/Variant to change its components/further css accordingly (if you don't supply the parameter it will set it to false underneath).

Reply:
`PaperProductPrices` needs to have the prop alright, but couldn’t you remove this prop and pass `true` to `PaperProductPrices` on line 171? It’s not really a big deal either way though, I think.

Reply:
Yes, understand, have tidied the unnecessary code, ta